PowerEditor/src/Parameters.cpp

```diff
@@ -1681,11 +1681,11 @@ bool NppParameters::getUserParametersFromXmlTree()
 std::pair<unsigned char, unsigned char> NppParameters::addUserDefineLangsFromXmlTree(TiXmlDocument *tixmldoc)
 {
 	if (!tixmldoc)
-		return std::pair<unsigned char, unsigned char>(0, 0);
+		return std::pair<unsigned char, unsigned char>('0', '0');
 
 	TiXmlNode *root = tixmldoc->FirstChild(TEXT("NotepadPlus"));
 	if (!root)
-		return std::pair<unsigned char, unsigned char>(0, 0);
+		return std::pair<unsigned char, unsigned char>('0', '0');

```

Comment:
in C/C++,  `0 != '0'`  -- This needs to be changed.  See inline comment above for details.